File: Apendice/22-MeAjude/MeAjude.pl
@d=split/ /,<>." ".<>;
$s="o\nMaior area: ";
$a=@d[0]*@d[1];
$b=@d[2]*@d[3];
if($a==$b){print"Empate";}else{print$a>$b?"Primeir".$s.$a:"Segund".$s.$b}

File: Apendice/22-MeAjude/MeAjude.py
i=raw_input
a,b,c,d=map(int,(i()+" "+i()).split())
s="o\nMaior area: "
e=a*b
f=c*d
print(("Segund"+s+`f`,"Primeir"+s+`e`)[e>f],"Empate")[e==f]
